aspire-orchestrator/SemanticWorkbench.Aspire.Hosting.Extensions/SemanticWorkbench.Aspire.Hosting.Extensions.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
One thing adding friction add increasing the overall complexity, is the number of "extras" to add to have Aspire. 
In this case there are three extra projects, vs a single project for the agent, that's quite a lot:
* app host
* app host extensions
* service defaults

what about merging the "app host extensions" project with "app host", reducing the number to two?

Reply:
Maybe we could move the Extensions in the libraries?